spec/integration/standard_tasks_spec.rb

```diff
@@ -58,13 +58,13 @@
     it 'converts odt' do
       content = fixture('heathen/ooword.odt').read
       new_content = converter.convert 'txt', content
-      expect(new_content.mime_type).to eq 'text/plain; charset=utf-8'
+      expect(new_content.mime_type).to eq 'text/plain; charset=us-ascii'
     end
 
     it 'converts docx' do
       content = fixture('heathen/msword.docx').read
       new_content = converter.convert 'txt', content
-      expect(new_content.mime_type).to eq 'text/plain; charset=utf-8'
+      expect(new_content.mime_type).to eq 'text/plain; charset=us-ascii'
```

Comment:
this is more or less OK for now, but the reason why the charset has changed should be investigated. It is likely it is a side effect of local configuration of the development environment, as such we should either ignore it, or to set the `LC_*` variables to known good values inside colore.

Anyway - cosmetic issues given our current priorities.